src/screens/ProfileScreen/index.js

```diff
@@ -18,11 +18,28 @@ export default class ProfileScreen extends Component {
   constructor(props) {
 		super(props);
 		this.state = {
-    language:" ",
-    cycle:" ",
-    last: " ",
+    language:"English",
+    cycle:"20 Days",
+    last: "14 July 2020",
 		};
-	}
+  }
+  
+  onPress = () => {
+    this.setState({
+      language:"Marathi",
+    })
+  }
+  onPress0 = () => {
+    this.setState({
+      cycle:"23 Days",
+    })
+  }
+  onPress1 = () => {
```

Comment:
similar here e.g `changeDate`. or `updateDate`
the function name should define its work